models/channel.go

```diff
@@ -0,0 +1,93 @@
+// pmm-managed
+// Copyright (C) 2017 Percona LLC
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+package models
+
+// ChannelType represents notificaion channel type.
+type ChannelType string
+
+// Available notificaion channel types.
+const (
+	Email     = ChannelType("email")
+	PagerDuty = ChannelType("pagerduty")
+	Slack     = ChannelType("slack")
+	WebHook   = ChannelType("webhook")
+)
+
+// Channel represents notification channel configuration.
+type Channel struct {
```

Comment:
Why you did not add `//reform:notification_channels` to generate the whole reform model? This way we don't have to write insert/update/delete helpers

Reply:
I used `Settings` as a reference, it doesn't use reform for some reason.

Reply:
Settings is a single-row table. That's unlike all other tables

Reply:
After discussion: refactored for using `reform`, table structure changed.